R/pool.R

```diff
@@ -164,6 +176,10 @@ Pool <- R6::R6Class("Pool",
           "returns the object to be pooled.")
       }
 
+      if (is.null(private$objClass)) {
+        private$objClass <- class(object)
```

Comment:
Knowing the class of the pooled objects is useful for printing.

This currently assumes that `minSize` is always >= 1, which seems like a reasonable assumption (but I could protect against it if you think it's important).

Reply:
Joe say: just double check this is ok (i.e. make a test).